integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java

```diff
@@ -304,6 +304,9 @@
   public static final boolean WEBLOGIC_SLIM =
       WEBLOGIC_IMAGE_TAG.contains("slim") ? true : false;
 
+  public static final boolean PSU_PATCH =
```

Comment:
Is there a reason to make this a boolean instead of a method with a meaningful name such as `isWebLogicPsuPatchApplied()` that returns a boolean? If so, I think the variable name should be changed to somethink like `psuPatchApplied `.

Reply:
Modified the boolean variable name to PSU_PATCH_APPLIED

Reply:
Created a utility method isWebLogicPsuPatchApplied() to get the WebLogic Image patch status